bin/create.js

```diff
@@ -37,6 +37,10 @@ const transformSVGToReactComponent = Promise.coroutine(function*(rawSVG, compone
 
   const viewBox = $svg.attr('viewBox');
 
+  if (viewBox !== `0 0 ${width} ${height}`) {
```

Comment:
If you already know the exact view box that should be added it might have been easier to just add that one instead of trying to extract it from the svg 😅

Reply:
I realised last minute that we always want a certain value to be set, and actually changed it from a regex test to a check if the value is an exact match.

Personally, I still prefer using the attribute though, purely for validation reasons.